frontend/src/pages/index.graphql

```diff
@@ -21,20 +21,25 @@ query IndexPage($code: String!, $language: String!) {
     gettingThereText: copy(key: "getting-there-text", language: $language)
 
     map {
+      id
+      image(width: 1280, height: 400, zoom: 15)
```

Comment:
Ok the `id` but what about this `image`? Is intended?

Reply:
Yes it is intended because I wanted to fetch everything together, but since we now have an ID I think I can remove it and have the MapWithLink run the query and thanks to the ID Apollo should be able to merge the two